src/Modules/Microsoft.Azure.Functions.PowerShellWorker/Microsoft.Azure.Functions.PowerShellWorker.psm1

```diff
@@ -130,3 +132,52 @@ function Push-OutputBinding {
         }
     }
 }
+
+<#
+.SYNOPSIS
+    Write the formated output of the pipeline object to the information stream before passing the object down to the pipeline.
+.DESCRIPTION
+    Write the formated output of the pipeline object to the information stream before passing the object down to the pipeline.
+.PARAMETER InputObject
+    The object from pipeline.
+.PARAMETER WriteToInformationChannel
+    Make the command write the pipeline object to the information stream.
+#>
+function Trace-PipelineObject {
```

Comment:
This is something that we run, right? The user wouldn't use this?

Reply:
Yes, it should be used only by us, but nothing prevents users from using it.

Reply:
Description is updated to call out that this function is for the worker to use only.